### ISSUE TITLE
From Azure Pipeline restore package succeed, but the content (css,js ,img) of package is not added in Web application .net

### ISSUE THREAD

**Issue by osmodies** [NuGet/Home#9073]
Remove the content above here and fill out details below.

## Details about Problem
From Azure Pipeline restore package succeed, but the content (css,js ,img) of package is not added in Web application .net
![image](https://user-images.githubusercontent.com/19317872/72763207-a4991480-3bb0-11ea-8be7-bde3405df93c.png)

NuGet product used (NuGet.exe | VS UI | Package Manager Console | dotnet.exe):

NuGet version :4.4.1

VS 2019

 Agent.OS Version | 10.0.17763 Microsoft Windows Server 2019 Standard (self Hosted Agent Azure Pipelines)

Agent.Version | 2.144.2

## Detailed repro steps so we can see the same problem

1.Execute Azure pipeline 
2. Restore Nuget package 
3. Build Solution: Faild, because it can't copy content (css,js,jpg,png) of package installed  in project web app C# VS 2019

## Other suggested things
in the Visual Studio Install package, but content don't copy in project. i must execute command Update-Package -reinstall in package manager console.

<details>
<summary>Verbose Logs</summary>
```
Log Task Nuget restore
##[section]Starting: NuGet restore
==============================================================================
Task         : NuGet
Description  : Restore, pack, or push NuGet packages, or run a NuGet command. Supports NuGet.org and authenticated feeds like Azure Artifacts and MyGet. Uses NuGet.exe and works with .NET Framework apps. For .NET Core and .NET Standard apps, use the .NET Core task.
Version      : 2.153.2
Author       : Microsoft Corporation
Help         : [More Information](https://go.microsoft.com/fwlink/?LinkID=613747)
==============================================================================
[command]C:\Windows\system32\chcp.com 65001
Active code page: 65001
Detected NuGet version 4.4.1.4656 / 4.4.1
SYSTEMVSSCONNECTION exists true
Saving NuGet.config to a temporary config file.
[command]D:\agent\_work\_tool\NuGet\4.4.1\ia32\nuget.exe sources Add -NonInteractive -Name 4803fc5b-18ee-4155-b47a-2b90237693e4 -Source https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/index.json -ConfigFile D:\agent\_work\26\Nuget\tempNuGet_358.config
Package Source with Name: 4803fc5b-18ee-4155-b47a-2b90237693e4 added successfully.
[command]D:\agent\_work\_tool\NuGet\4.4.1\ia32\nuget.exe sources Add -NonInteractive -Name NuGetOrg -Source https://api.nuget.org/v3/index.json -ConfigFile D:\agent\_work\26\Nuget\tempNuGet_358.config
Package Source with Name: NuGetOrg added successfully.
Saving NuGet.config to a temporary config file.
[command]D:\agent\_work\_tool\NuGet\4.4.1\ia32\nuget.exe restore D:\agent\_work\26\s\WAP-EdocsPiden.sln -NoCache -Verbosity Detailed -NonInteractive -ConfigFile D:\agent\_work\26\Nuget\tempNuGet_358.config
NuGet Version: 4.4.1.4656
MSBuild auto-detection: using msbuild version '16.3.2.50909' from 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\bin'. Use option -MSBuildVersion to force nuget to use a specific version of MSBuild.
MSBuild P2P timeout [ms]: 120000
C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\bin\msbuild.exe "C:\Users\user_devops\AppData\Local\Temp\NuGetScratch\nmloz5is.2q3.nugetinputs.targets" /t:GenerateRestoreGraphFile /nologo /nr:false /v:q /p:NuGetRestoreTargets="C:\Users\user_devops\AppData\Local\Temp\NuGetScratch\l04r3rkk.zkb.nugetrestore.targets" /p:RestoreTaskAssemblyFile="D:\agent\_work\_tool\NuGet\4.4.1\ia32\nuget.exe" /p:RestoreSolutionDirectory="D:\agent\_work\26\s" /p:RestoreConfigFile="D:\agent\_work\26\Nuget\tempNuGet_358.config" /p:SolutionDir="D:\agent\_work\26\s"

Restoring NuGet package ComponentesCharts.1.0.9.
Restoring NuGet package ComponentesCargueArchivos.1.0.1.
Restoring NuGet package ComponentesControlesEdesk.1.0.10.
Restoring NuGet package ComponentesEstaticosBase.1.0.13.
Restoring NuGet package ComponentesPlantillaEdeskBase.1.1.5.
Restoring NuGet package ControlesEdesk.1.0.10.
Restoring NuGet package EdeskManager.1.2.8.
Restoring NuGet package Ghostscript.NET.1.2.1.
Restoring NuGet package iTextSharp.5.5.13.1.
Restoring NuGet package Microsoft.CodeDom.Providers.DotNetCompilerPlatform.2.0.1.
Restoring NuGet package Newtonsoft.Json.12.0.2.
Restoring NuGet package NPOI.2.1.3.
Restoring NuGet package SharpZipLib.0.86.0.
Restoring NuGet package TokenSYC.1.1.0.
Restoring NuGet package UtilSyc.1.0.1.
  GET https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/componentescarguearchivos/1.0.1/componentescarguearchivos.1.0.1.nupkg
  GET https://api.nuget.org/v3-flatcontainer/componentescarguearchivos/1.0.1/componentescarguearchivos.1.0.1.nupkg
  GET https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/componentescharts/1.0.9/componentescharts.1.0.9.nupkg
  GET https://api.nuget.org/v3-flatcontainer/componentescharts/1.0.9/componentescharts.1.0.9.nupkg
  GET https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/componentescontrolesedesk/1.0.10/componentescontrolesedesk.1.0.10.nupkg
  GET https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/componentesestaticosbase/1.0.13/componentesestaticosbase.1.0.13.nupkg
  GET https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/componentesplantillaedeskbase/1.1.5/componentesplantillaedeskbase.1.1.5.nupkg
  GET https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/controlesedesk/1.0.10/controlesedesk.1.0.10.nupkg
  GET https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/edeskmanager/1.2.8/edeskmanager.1.2.8.nupkg
  GET https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/ghostscript.net/1.2.1/ghostscript.net.1.2.1.nupkg
  GET https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/itextsharp/5.5.13.1/itextsharp.5.5.13.1.nupkg
  GET https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/microsoft.codedom.providers.dotnetcompilerplatform/2.0.1/microsoft.codedom.providers.dotnetcompilerplatform.2.0.1.nupkg
  GET https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/newtonsoft.json/12.0.2/newtonsoft.json.12.0.2.nupkg
  GET https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/npoi/2.1.3/npoi.2.1.3.nupkg
  GET https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/sharpziplib/0.86.0/sharpziplib.0.86.0.nupkg
  GET https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/tokensyc/1.1.0/tokensyc.1.1.0.nupkg
  GET https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/utilsyc/1.0.1/utilsyc.1.0.1.nupkg
  GET https://api.nuget.org/v3-flatcontainer/componentescontrolesedesk/1.0.10/componentescontrolesedesk.1.0.10.nupkg
  GET https://api.nuget.org/v3-flatcontainer/componentesestaticosbase/1.0.13/componentesestaticosbase.1.0.13.nupkg
  GET https://api.nuget.org/v3-flatcontainer/componentesplantillaedeskbase/1.1.5/componentesplantillaedeskbase.1.1.5.nupkg
  GET https://api.nuget.org/v3-flatcontainer/controlesedesk/1.0.10/controlesedesk.1.0.10.nupkg
  GET https://api.nuget.org/v3-flatcontainer/edeskmanager/1.2.8/edeskmanager.1.2.8.nupkg
  GET https://api.nuget.org/v3-flatcontainer/ghostscript.net/1.2.1/ghostscript.net.1.2.1.nupkg
  GET https://api.nuget.org/v3-flatcontainer/itextsharp/5.5.13.1/itextsharp.5.5.13.1.nupkg
  GET https://api.nuget.org/v3-flatcontainer/microsoft.codedom.providers.dotnetcompilerplatform/2.0.1/microsoft.codedom.providers.dotnetcompilerplatform.2.0.1.nupkg
  GET https://api.nuget.org/v3-flatcontainer/newtonsoft.json/12.0.2/newtonsoft.json.12.0.2.nupkg
  GET https://api.nuget.org/v3-flatcontainer/npoi/2.1.3/npoi.2.1.3.nupkg
  GET https://api.nuget.org/v3-flatcontainer/sharpziplib/0.86.0/sharpziplib.0.86.0.nupkg
  GET https://api.nuget.org/v3-flatcontainer/tokensyc/1.1.0/tokensyc.1.1.0.nupkg
  GET https://api.nuget.org/v3-flatcontainer/utilsyc/1.0.1/utilsyc.1.0.1.nupkg
CredentialProvider.TeamBuild: "D:\agent\_work\_tasks\NuGetCommand_333b11bd-d341-40d9-afcf-b32d5ce6f23b\2.153.2\CredentialProvider\CredentialProvider.TeamBuild.exe" -uri https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/index.json -nonInteractive -verbosity detailed
CredentialProvider.TeamBuild: URI Prefixes:
CredentialProvider.TeamBuild:     https://devops.
CredentialProvider.TeamBuild: URI: https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/index.json
CredentialProvider.TeamBuild: Is retry: False
CredentialProvider.TeamBuild: Matched prefix: https://devops.
  OK https://api.nuget.org/v3-flatcontainer/itextsharp/5.5.13.1/itextsharp.5.5.13.1.nupkg 226ms
  OK https://api.nuget.org/v3-flatcontainer/ghostscript.net/1.2.1/ghostscript.net.1.2.1.nupkg 250ms
  OK https://api.nuget.org/v3-flatcontainer/microsoft.codedom.providers.dotnetcompilerplatform/2.0.1/microsoft.codedom.providers.dotnetcompilerplatform.2.0.1.nupkg 250ms
  OK https://api.nuget.org/v3-flatcontainer/npoi/2.1.3/npoi.2.1.3.nupkg 250ms
  OK https://api.nuget.org/v3-flatcontainer/sharpziplib/0.86.0/sharpziplib.0.86.0.nupkg 250ms
  OK https://api.nuget.org/v3-flatcontainer/newtonsoft.json/12.0.2/newtonsoft.json.12.0.2.nupkg 270ms
  NotFound https://api.nuget.org/v3-flatcontainer/componentesestaticosbase/1.0.13/componentesestaticosbase.1.0.13.nupkg 921ms
  NotFound https://api.nuget.org/v3-flatcontainer/controlesedesk/1.0.10/controlesedesk.1.0.10.nupkg 920ms
  NotFound https://api.nuget.org/v3-flatcontainer/edeskmanager/1.2.8/edeskmanager.1.2.8.nupkg 921ms
  NotFound https://api.nuget.org/v3-flatcontainer/componentescontrolesedesk/1.0.10/componentescontrolesedesk.1.0.10.nupkg 922ms
  NotFound https://api.nuget.org/v3-flatcontainer/componentescarguearchivos/1.0.1/componentescarguearchivos.1.0.1.nupkg 956ms
  NotFound https://api.nuget.org/v3-flatcontainer/componentesplantillaedeskbase/1.1.5/componentesplantillaedeskbase.1.1.5.nupkg 922ms
  NotFound https://api.nuget.org/v3-flatcontainer/tokensyc/1.1.0/tokensyc.1.1.0.nupkg 920ms
  NotFound https://api.nuget.org/v3-flatcontainer/utilsyc/1.0.1/utilsyc.1.0.1.nupkg 920ms
  NotFound https://api.nuget.org/v3-flatcontainer/componentescharts/1.0.9/componentescharts.1.0.9.nupkg 927ms
  OK https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/utilsyc/1.0.1/utilsyc.1.0.1.nupkg 1435ms
  OK https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/componentesestaticosbase/1.0.13/componentesestaticosbase.1.0.13.nupkg 1438ms
  OK https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/componentescarguearchivos/1.0.1/componentescarguearchivos.1.0.1.nupkg 1470ms
  OK https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/edeskmanager/1.2.8/edeskmanager.1.2.8.nupkg 1437ms
  OK https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/controlesedesk/1.0.10/controlesedesk.1.0.10.nupkg 1438ms
  OK https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/componentescharts/1.0.9/componentescharts.1.0.9.nupkg 1465ms
  OK https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/componentesplantillaedeskbase/1.1.5/componentesplantillaedeskbase.1.1.5.nupkg 1439ms
  OK https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/sharpziplib/0.86.0/sharpziplib.0.86.0.nupkg 1438ms
  OK https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/ghostscript.net/1.2.1/ghostscript.net.1.2.1.nupkg 1439ms
  OK https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/componentescontrolesedesk/1.0.10/componentescontrolesedesk.1.0.10.nupkg 1441ms
  OK https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/newtonsoft.json/12.0.2/newtonsoft.json.12.0.2.nupkg 1439ms
  OK https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/itextsharp/5.5.13.1/itextsharp.5.5.13.1.nupkg 1440ms
  OK https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/npoi/2.1.3/npoi.2.1.3.nupkg 1439ms
  OK https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/microsoft.codedom.providers.dotnetcompilerplatform/2.0.1/microsoft.codedom.providers.dotnetcompilerplatform.2.0.1.nupkg 1440ms
  OK https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/flat2/tokensyc/1.1.0/tokensyc.1.1.0.nupkg 1440ms
Adding package 'TokenSYC.1.1.0' to folder 'D:\agent\_work\26\s\packages'
Adding package 'Ghostscript.NET.1.2.1' to folder 'D:\agent\_work\26\s\packages'
Adding package 'ComponentesCargueArchivos.1.0.1' to folder 'D:\agent\_work\26\s\packages'
Adding package 'UtilSyc.1.0.1' to folder 'D:\agent\_work\26\s\packages'
Added package 'TokenSYC.1.1.0' to folder 'D:\agent\_work\26\s\packages'
Added package 'TokenSYC.1.1.0' to folder 'D:\agent\_work\26\s\packages' from source 'https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/index.json'
Adding package 'ControlesEdesk.1.0.10' to folder 'D:\agent\_work\26\s\packages'
Added package 'UtilSyc.1.0.1' to folder 'D:\agent\_work\26\s\packages'
Added package 'UtilSyc.1.0.1' to folder 'D:\agent\_work\26\s\packages' from source 'https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/index.json'
Added package 'Ghostscript.NET.1.2.1' to folder 'D:\agent\_work\26\s\packages'
Added package 'Ghostscript.NET.1.2.1' to folder 'D:\agent\_work\26\s\packages' from source 'https://api.nuget.org/v3/index.json'
Adding package 'EdeskManager.1.2.8' to folder 'D:\agent\_work\26\s\packages'
Added package 'ControlesEdesk.1.0.10' to folder 'D:\agent\_work\26\s\packages'
Added package 'ControlesEdesk.1.0.10' to folder 'D:\agent\_work\26\s\packages' from source 'https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/index.json'
Added package 'EdeskManager.1.2.8' to folder 'D:\agent\_work\26\s\packages'
Added package 'EdeskManager.1.2.8' to folder 'D:\agent\_work\26\s\packages' from source 'https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/index.json'
Added package 'ComponentesCargueArchivos.1.0.1' to folder 'D:\agent\_work\26\s\packages'
Added package 'ComponentesCargueArchivos.1.0.1' to folder 'D:\agent\_work\26\s\packages' from source 'https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/index.json'
Adding package 'ComponentesCharts.1.0.9' to folder 'D:\agent\_work\26\s\packages'
Adding package 'SharpZipLib.0.86.0' to folder 'D:\agent\_work\26\s\packages'
Added package 'SharpZipLib.0.86.0' to folder 'D:\agent\_work\26\s\packages'
Added package 'SharpZipLib.0.86.0' to folder 'D:\agent\_work\26\s\packages' from source 'https://api.nuget.org/v3/index.json'
Adding package 'ComponentesEstaticosBase.1.0.13' to folder 'D:\agent\_work\26\s\packages'
Added package 'ComponentesCharts.1.0.9' to folder 'D:\agent\_work\26\s\packages'
Added package 'ComponentesCharts.1.0.9' to folder 'D:\agent\_work\26\s\packages' from source 'https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/index.json'
Adding package 'iTextSharp.5.5.13.1' to folder 'D:\agent\_work\26\s\packages'
Adding package 'Newtonsoft.Json.12.0.2' to folder 'D:\agent\_work\26\s\packages'
Added package 'iTextSharp.5.5.13.1' to folder 'D:\agent\_work\26\s\packages'
Added package 'iTextSharp.5.5.13.1' to folder 'D:\agent\_work\26\s\packages' from source 'https://api.nuget.org/v3/index.json'
Added package 'ComponentesEstaticosBase.1.0.13' to folder 'D:\agent\_work\26\s\packages'
Added package 'ComponentesEstaticosBase.1.0.13' to folder 'D:\agent\_work\26\s\packages' from source 'https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/index.json'
Adding package 'ComponentesPlantillaEdeskBase.1.1.5' to folder 'D:\agent\_work\26\s\packages'
Added package 'Newtonsoft.Json.12.0.2' to folder 'D:\agent\_work\26\s\packages'
Added package 'Newtonsoft.Json.12.0.2' to folder 'D:\agent\_work\26\s\packages' from source 'https://api.nuget.org/v3/index.json'
Adding package 'NPOI.2.1.3' to folder 'D:\agent\_work\26\s\packages'
Adding package 'ComponentesControlesEdesk.1.0.10' to folder 'D:\agent\_work\26\s\packages'
Added package 'NPOI.2.1.3' to folder 'D:\agent\_work\26\s\packages'
Added package 'NPOI.2.1.3' to folder 'D:\agent\_work\26\s\packages' from source 'https://api.nuget.org/v3/index.json'
Added package 'ComponentesPlantillaEdeskBase.1.1.5' to folder 'D:\agent\_work\26\s\packages'
Added package 'ComponentesPlantillaEdeskBase.1.1.5' to folder 'D:\agent\_work\26\s\packages' from source 'https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/index.json'
Added package 'ComponentesControlesEdesk.1.0.10' to folder 'D:\agent\_work\26\s\packages'
Added package 'ComponentesControlesEdesk.1.0.10' to folder 'D:\agent\_work\26\s\packages' from source 'https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/index.json'
Adding package 'Microsoft.CodeDom.Providers.DotNetCompilerPlatform.2.0.1' to folder 'D:\agent\_work\26\s\packages'
Added package 'Microsoft.CodeDom.Providers.DotNetCompilerPlatform.2.0.1' to folder 'D:\agent\_work\26\s\packages'
Added package 'Microsoft.CodeDom.Providers.DotNetCompilerPlatform.2.0.1' to folder 'D:\agent\_work\26\s\packages' from source 'https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/index.json'

NuGet Config files used:
    D:\agent\_work\26\Nuget\tempNuGet_358.config

Feeds used:
    https://devops._packaging/4803fc5b-18ee-4155-b47a-2b90237693e4/nuget/v3/index.json
    https://api.nuget.org/v3/index.json

Installed:
    15 package(s) to packages.config projects
##[section]Finishing: NuGet restore
----=======
Task Build Solution
##[error]C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VisualStudio\v16.0\Web\Microsoft.Web.Publishing.targets(2601,5): Error : Copying file recursosED\css\img\bgModal.png to obj\Release\Package\PackageTmp\recursosED\css\img\bgModal.png failed. Could not find a part of the path 'recursosED\css\img\bgModal.png'.

this content is of package Nuget preview installed.

```
</details>

**Comment by rrelyea** [comment]
What kind of project are you hosting the package in?
Are you using packagereference or packages.config?
Can you reproduce the problem on your dev machine too?
Can you give us a small repro project/solution that shows us the problem?
Has this ever worked? What changed since then?

**Comment by osmodies** [comment]
What kind of project are you hosting the package in?
- Web Application .Net Framework
Are you using packagereference or packages.config?
- packages.config
Can you reproduce the problem on your dev machine too?
- Yes, Local i must update package-reinstall from Pakage manager Console

**Comment by nkolev92** [comment]
@osmodies I'm unable to repro any content files installation issues with VS 2019.
Is this something for which you have a reliable repro?
Can you try to record the repro for us or provide a repro project?

**Comment by osmodies** [comment]
Hi, Excuse me for my english 

1. Attach nuspec of create package
[PipelineComponenteestaticobase.txt](https://github.com/NuGet/Home/files/4517084/PipelineComponenteestaticobase.txt)
[ComponentesEstaticosBase.nuspec.txt](https://github.com/NuGet/Home/files/4517085/ComponentesEstaticosBase.nuspec.txt)

2. After add in web applicacion the package created and published in Azure Artifacts
![image](https://user-images.githubusercontent.com/19317872/80002002-aeb70180-8484-11ea-9075-73416a91fa02.png)

3. when i exeucte build f web application failed because no add content in web app in the moment of publish, despite that tak of nuger restore is success 
![image](https://user-images.githubusercontent.com/19317872/80002781-b4611700-8485-11ea-94a3-be5fee6ea81a.png)

##[error]C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VisualStudio\v16.0\Web\Microsoft.Web.Publishing.targets(2601,5): Error : Copying file recursosED\css_min\bootstrap.min.css to obj\Release\Package\PackageTmp\recursosED\css_min\bootstrap.min.css failed. Could not find a part of the path 'recursosED\css_min\bootstrap.min.css'.

My workaorund was creating script powr shell that copy content restore in rout of web application>
$sourceDir = "$(system.defaultworkingdirectory)" + "\packages"
$targetDir = "$(system.defaultworkingdirectory)"
$filter= "recursosED"
$bin = Get-ChildItem -Recurse -Path $sourceDir | Where-Object {$_.Name -match $filter} | % { $_.FullName }

foreach ($item in $bin) { 
    
    If(!(test-path $targetDir))
        {
            New-Item -ItemType Directory -Force -Path $targetDir
        }
    Copy-Item -Path $item -Destination $targetDir -Recurse -Force
}

Thanks...

**Comment by nkolev92** [comment]
Hey @osmodies 

In packages.config the addition of content files happens at install time. 

If you check your project file, can you observe any references to the content files in question?
If not, can you please run `Update-Package -Reinstall` on the project in question. 
Alternatively you can uninstall and install this specific package.

**Comment by nkolev92** [comment]
Closing per the above comment. 
This is currently the expected design of packages.config based projects.

**Comment by osmodies** [comment]
Thanks, for your answer, but in the installation in azure pipelines not add content files to solution, for this i used this script powershell
My workaorund was creating script powr shell that copy content restore in rout of web application>
$sourceDir = "$(system.defaultworkingdirectory)" + "\packages"
$targetDir = "$(system.defaultworkingdirectory)"
$filter= "recursosED"
$bin = Get-ChildItem -Recurse -Path $sourceDir | Where-Object {$.Name -match $filter} | % { $.FullName }

foreach ($item in $bin) {

If(!(test-path $targetDir))
    {
        New-Item -ItemType Directory -Force -Path $targetDir
    }
Copy-Item -Path $item -Destination $targetDir -Recurse -Force.

Considering that in CI when restore package it install but no add content.

**Comment by zivkan** [comment]
I just tried very quickly to reproduce your scenario, but I was not able to reproduce your result.

I created an empty folder `contentPkg`, ran `dotnet new classlib`, created a file `context.xml`, and added `<Content Include="content.xml" PackageCopyToOutput="true" />` in an `ItemGroup`. I then ran `dotnet pack` and copied the nupkg to a local package source.

In another folder I ran `dotnet new mvc`, then `dotnet add package contentPkg`. Finally, I ran `dotnet publish`, and now when I run `dir bin\Debug\publish`, I can see `cotentPkg.dll` and `context.xml`.

Therefore, it looks to me like everything is working as designed.
